src/screens/SettingsScreen.js

```diff
@@ -132,6 +140,42 @@ export const SettingsScreen = () => {
         });
       }
 
+      if (settings.onboarding) {
+        const onboarding = await readFromStore(ONBOARDING_STORE_KEY);
+
+        additionalSectionedData.push({
+          data: [
+            {
+              title: texts.settingsTitles.onboarding,
+              topDivider: false,
```

Comment:
as this is the third entry, a top divider is necessary to separate from matomo setting.

Reply:
a7587b4e5e398ac83c0cd01a398797166b4d72b2